src/app/api/documents/route.ts

```
@@ -0,0 +1,207 @@
+          error: `Invalid 'page_size' parameter. Must be between 1 and ${maxPageSize}.`,
+        },
+        { status: 400 }
+      );
+    }
+    const since = searchParams.get('since'); // date-time string (optional)
+
+    // Build Mongo filter
+    const filter: Record<string, any> = {};
+    if (since) {
+      const sinceDate = new Date(since);
+      if (isNaN(sinceDate.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)